pkg/controller/storagecluster/uninstall_reconciler.go

```diff
@@ -356,12 +357,12 @@ func (r *ReconcileStorageCluster) deleteResources(sc *ocsv1.StorageCluster, reqL
 		return err
 	}
 
-	err = r.deleteCephObjectStores(sc, reqLogger)
```

Comment:
Are you intentionally removing the call to this function?

Reply:
Sorry for the distraction @jarrpa . I am debugging something with this PR. It is not ready yet. I will mark it WIP.